src/features/intake/components/EmbedCodeBlock.tsx

```
@@ -0,0 +1,213 @@
+    document.body.appendChild(textarea);
+    try {
+      textarea.select();
+      const copied = document.execCommand('copy');
+      if (copied) {
+        onSuccess();
+      } else {
+        onError('Clipboard is not available.');
+      }
+    } catch (error) {
+      throw error;
+    } finally {
```

ESLint

[error] Unnecessary catch clause.

(no-useless-catch)